src/server/KbnServer.js

```diff
@@ -83,4 +83,18 @@ module.exports = class KbnServer {
   async close() {
     await fromNode(cb => this.server.stop(cb));
   }
+
+  async inject(opts) {
+    if (!this.server) await this.ready();
+
+    return await fromNode(cb => {
+      try {
```

Comment:
No, it's not necessary, but I've gotten into the habit of doing so because it feels like the way that async/await was designed to be used. I also prefer not to return a bluebird promise some of the time and a basic promise other times.